### DISCUSSION TITLE
Have anyone tried automating the mobile and web app in same project using webdio multiremote?

### DISCUSSION THREAD

**Discussion by surajb0710** [webdriverio/webdriverio#10479] [Q&A]
I am able to run the mobile app and web app separately by commenting out capability for one and running the test for another. But, if I am enabling both of the capabilities, then it messes up.
.
I just need to open the app in mobile and url in web browser at same time, rest I can figure out.
.
I am getting diff errors each time and most common error is platform name capability is missing, even though I have added it.

**Comment by christian-bromann** [comment]
@surajb0710 please provide error logs and detailed information about your setup, so folks have information to help you out.
